docs/modules/ROOT/pages/explanations/backup.adoc

```diff
@@ -0,0 +1,38 @@
+= Keycloak Backup
+
+There are two possibilities to backup Keycloak.
+The first approach is backup the database, which has been implemented using the K8up tool.
+The second way is to export all objects via the Keycloak application.
+This hasn't been implemented, but a potential future way to handle backups.
+
+== Database backup
+
+Most of the sources are recommending doing a database backup:
+
+* https://www.keycloak.org/docs/latest/upgrading/#_prep_migration[Handbook]
+* https://www.keycloak.org/docs/latest/server_installation/index.html#_backup-cr[Operator]
+* https://keycloak.discourse.group/t/best-practice-for-backing-up-the-db/4811[Forum]
+
+The handbook also mentioning configuration, themes, scripts require a backup.
+In the case of a container deployment, those parts are attached to the container and so will usually be part of the deployment.
+This component doesn't take care to backup something else than the built-in database.
+
+== Export and import
+
+While this would be the preferable way to implement the backup, the approach has currently some downsides:
+
+About the export/import the handbook states:
+
+[quote,'Keycloak Handbook, https://www.keycloak.org/docs/latest/server_admin/#_export_import[Export and Import]']
+____
+It's important to note that because import and export happens at server startup, no other actions should be taken on the server or the database while this happens.
+____
+Doesn't sounds like there is currently a lock perventing paralell access during the export and guarantee those consistency.
+
+[quote,'Keycloak Handbook, https://www.keycloak.org/docs/latest/server_admin/#admin-console-export-import[Admin console export/import]']
+____
+Attributes containing secrets or private information will be masked in export file. Export files obtained via Admin Console are thus not appropriate for backups or data transfer between servers. Only boot-time exports are appropriate for that.
+____
+Stop the service doing a export on startup, sounds not like a valuable solution, as we don't want to restart the service for that purpose.
+
+Reconsider using this kind of backup in newer versions, once the export functionality might have been improved.
```

Comment:
I'm not sure whether the Antora documentation is the right place to explain a not-implemented feature or situation.
Personally, I'd create a new GitHub Issue for this. Maybe the issue could be linked, but documenting this here prevents possible discussions.

@simu @corvus-ch WDYT?

Reply:
I thought that was what the explanation section was for?

Reply:
Well, I'd regard explanation section to explain things why they were implemented as such.
Here however, export/import is not implemented at all and has no relevance in the component. To me it's like discussing possible a feature request resp. story.

But I don't really have hard feelings if it's in there either.

Reply:
I'm open to remove this part and create an feature request instead. I guess it depends how likely we are going to work on that, because my original intention was to document my findings somewhere.